docs/development/documentation_guidelines.rst

```diff
@@ -25,6 +25,8 @@ To get started editing the docs:
 
       pip install --no-deps --require-hashes -r securedrop/requirements/python3/develop-requirements.txt
 
+.. include:: ../includes/virtualenv.txt   
```

Comment:
The include directive breaks up the automatic numbering via the `#.` syntax (the 3rd item is again restarted at 1). You can fix that by just manually numbering the steps here, i.e. replacing `#.` with `1.`, `2.`, `3.`. You can verify the result of such changes by using the `make docs` autobuild and visiting http://localhost:8000/